### PR TITLE
chore: add an end to end parsing test

### PR DESCRIPTION
**Pull request by srijan-deepsource**
Changes:
  - Add end to end parsing test
  - Fix assets being copied in the Dockerfile
  - Explicitly log the info passed to sentry.raise_info
  - Add instruction about adding fixture for parsing test in readme

### CODE REVIEW

**Comment by srijan-deepsource** [comment]
Test coverage failing for missing lines in sentry.py. Not critical.

File: sarif-parser/src/sarif_parser/__init__.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 import os.path
 from typing import Any, TypedDict
 
+import sentry
```
Comment (tushar-deepsource) [inline sarif-parser/src/sarif_parser/__init__.py:8]: Unused?

File: tests/test_report_parsing.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Any, Dict, Iterator
+
+import run_community_analyzer
+
+
+def make_artifact(report_path: str, workdir: str = "") -> str:
+    """Return an artifact file for the given report, with the given workdir."""
+    artifact_path = tempfile.NamedTemporaryFile().name
```
Comment (tushar-deepsource) [inline tests/test_report_parsing.py:13]: we should be using pytest's `tmp_path`, but this is ok for now.

**Review by tushar-deepsource — COMMENTED** [review]
two comments, LGTM otherwise.

File: sarif-parser/src/sarif_parser/__init__.py (hunk 1 above)
Comment (srijan-deepsource) [reply sarif-parser/src/sarif_parser/__init__.py:8]: https://github.com/DeepSourceCorp/community-analyzers/pull/10/files/5b5b11abf9b223cd188a9ea0cb280bbfe8144129#diff-c74668d6ffa071405b267954f825ba8087aef9ed47ac8121e9f65f5c61540785R117

**Comment by srijan-deepsource** [comment]
Found a bug. We aren't installing sentry_sdk in the dockerfile. Fixing.

**Review by tushar-deepsource — APPROVED** [review]

**Review by tushar-deepsource — APPROVED** [review]

**Pull request merged**